dspace/config/spring/api/scripts.xml

```diff
@@ -108,4 +108,13 @@
         <property name="dspaceRunnableClass" value="org.dspace.app.suggestion.runnable.PublicationLoaderRunnableCli"/>
     </bean>
 
+    <bean id="dspace-object-deletion" class="org.dspace.deletion.process.DSpaceObjectDeletionProcessScriptConfiguration" primary="true">
```

Comment:
Since none of our scripts use dspace in the name, I'd recommend we call this either object-deletion or just delete-object.